pom.xml

```diff
@@ -147,6 +159,7 @@
 					<imageName>${project.artifactId}</imageName>
 					<buildArgs>
 						-H:+AddAllCharsets
+						-H:+TraceSecurityServices
```

Comment:
According to [JCA Security Services in Native Image](https://www.graalvm.org/22.0/reference-manual/native-image/JCASecurityServices/#securerandom), we should probably use `--initialize-at-run-time=...` for `SecureRandom`.